cafe/models.py

```diff
@@ -0,0 +1,236 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from cafe.utils import calculation_salary
+
+
+class Cafe(models.Model):
+    """
+    Cafe model with name field.
+    """
+
+    name = models.CharField(max_length=63, unique=True)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        ordering = ["id"]
+
+
+class Barista(models.Model):
+    """
+    Barista model with full_name field.
+    """
+
+    full_name = models.CharField(max_length=150, blank=False)
+
+    def __str__(self):
+        return self.full_name
+
+    class Meta:
+        ordering = ["id"]
+
+
+class Rate(models.Model):
+    """
+    Barista rates model. Based on these data, it is estimated how much a
+    barista earned per day in a certain cafe.
+    The rate has three main parameters:
+
+    'min_wage' - the minimum amount that the barista will receive if the cafe’s
+    income that day is less than this parameter.
+
+    'percent' - this is the percentage of the cafe’s income that the barista
+    will receive if the cafe’s daily income is higher than the 'min_rate'.
+
+    'additive' - an additional amount to the barista’s salary if the cafe’s
+    daly income is higher than the minimum rate.
+
+    Each barista have only one rate for each cafe.
+    """
+
+    min_wage = models.IntegerField(blank=False)
```

Comment:
In case of money it's better to use DecimalFiel
See https://dev.to/koladev/django-tip-use-decimalfield-for-money-3f63 for example

Reply:
Yes, I know that it is better to use DecimalField for money. But this is not a shop, so I decided that I could ignore the accuracy. In general, according to the idea of the project, it is satisfied with an integer number. I even wrote about this in the description of the function that calculates salaries. I'll add this information to the project's readme file.

Reply:
Yes, I know that it is better to use DecimalField for money. But this is not a shop, so I decided that I could ignore the accuracy. In general, according to the idea of the project, it is satisfied with an integer number. I even wrote about this in the description of the function that calculates salaries. I'll add this information to the project's readme file.